modules/react/breadcrumbs/lib/BreadcrumbsList.tsx

```diff
@@ -6,9 +6,8 @@ import {useOverflowListMeasure, useListRenderItems} from '@workday/canvas-kit-re
 import {space} from '@workday/canvas-kit-react/tokens';
 import {useBreadcrumbsModel} from './hooks/useBreadcrumbsModel';
 
-// Use `Partial` here to make `spacing` optional
 export interface BreadcrumbsListProps<T = any>
-  extends Omit<Partial<ExtractProps<typeof Flex, never>>, 'children'> {
+  extends Omit<ExtractProps<typeof Flex, never>, 'children'> {
```

Comment:
same here, i think you can simplify this type to extend Flex without omitting children